sympy/calculus/tests/test_util.py

```diff
@@ -446,9 +448,8 @@ def test_AccumBounds_pow():
 
     assert AccumBounds(-4, 2)**Rational(2, 3) == AccumBounds(0, 2*2**Rational(1, 3))
 
-    assert AccumBounds(-1, 5)**S.Half == AccumBounds(0, sqrt(5))
-    assert AccumBounds(-oo, 2)**S.Half == AccumBounds(0, sqrt(2))
-    assert AccumBounds(-2, 3)**Rational(-1, 4) == AccumBounds(0, oo)
+    assert AccumBounds(-1, 0)**S.Half is S.NaN
+    assert AccumBounds(-2, 3)**Rational(-1, 4) is S.ComplexInfinity
```

Comment:
This could be `S.NaN`. The values may be finite but unbounded.

Reply:
I was thinking that they may be trumped by the `1/0`.